server/tests/test_repository_postgres.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+import os
+import unittest
+from functools import wraps
+
+import testing.postgresql
+
+from tests.utils import standard_seed_db
+from app.repository import Observation, ObservationFile, ObservationFileId, ObservationId, Repository, SatelliteId, StationId
+
+Postgresql: testing.postgresql.PostgresqlFactory
+
+def setUpModule():
```

Comment:
Great tests! I love them. I've pushed a bunch of extra tests that are a bit more thorough.